pandas/tests/io/test_html.py

```diff
@@ -134,28 +134,26 @@ def test_to_html_compat(self):
         res = self.read_html(out, attrs={"class": "dataframe"}, index_col=0)[0]
         tm.assert_frame_equal(res, df)
 
-    @pytest.mark.xfail(reason="Html file was removed")
     @tm.network
     def test_banklist_url_positional_match(self):
-        url = "https://www.fdic.gov/bank/individual/failed/banklist.html"
```

Comment:
Think this just redirects to the same as the below?

Reply:
Yeah, we shouldn't rely on them to keep redirecting it for us, so I changed it.

Reply:
thanks!